src/ec/packages/ec-component-site-header/site-header.story.js

```diff
@@ -0,0 +1,56 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs } from '@storybook/addon-knobs';
+import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import withCode from '@ecl-twig/storybook-addon-code';
+
+import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
+
+import siteHeaderDocs from './docs/site-header.md';
+import siteHeader from './site-header.html.twig';
+
+storiesOf('Components/Site Header', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withNotes)
+  .addDecorator(withCode)
+  .add(
+    'default',
+    () =>
+      siteHeader({
+        icon_file_path: defaultSprite,
+        header_link: {
+          url: '/example-1',
+        },
+      }),
+    {
+      notes: { markdown: siteHeaderDocs },
+    }
+  )
+  .add(
+    'translated',
+    () =>
+      siteHeader({
+        icon_file_path: defaultSprite,
+        language: {
+          url: '/example',
+          code: 'fr',
+          label: 'Français',
+        },
+        header_link: {
+          url: '/example-1',
+          aria_label: 'Commmission Européenne',
+        },
+        header_image: {
+          src: 'static/media/logo--fr.a8aaa7ab.svg',
```

Comment:
you can find the logos in package `@ecl/ec-resources-logo`: https://github.com/ec-europa/europa-component-library/tree/v2-dev/src/systems/ec/resources/logo

Reply:
> you can find the logos in package `@ecl/ec-resources-logo`: https://github.com/ec-europa/europa-component-library/tree/v2-dev/src/systems/ec/resources/logo

I have already made changes locally but I need help.

Could you tell me how to correctly configure the `webpack.config.js` file so that svg files are not compiled to one `icon.svg` but separately - so that I can implement them in the `Site Header`.

Now, when I build the EC environment, I have one svg file (`icon.svg`), the content of which is equal to the last imported svg, eg from such a resource: `import headerFrenchLogo from @ecl/ec-resources-logo/logo-fr.svg`.

Thank you for your help.

Reply:
it will be fixed in my PR on language list, but you can already apply the changes if you want: https://github.com/ec-europa/ecl-twig/pull/27/files#diff-ed535e5276105399f01342171198a43eR19

Reply:
Changed in https://github.com/ec-europa/ecl-twig/pull/33/commits/7fac7df766c33c18932199083bc08b9ef656e676